code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_powers.dm

```diff
@@ -780,6 +780,9 @@
 	if (!check_and_use_plasma_owner())
 		return
 
+	if (!isxeno(target))
+		return
+
```

Comment:
Why add a redundant check? You're checking both xeno and human immediately after this.